admin/index.php

```diff
@@ -122,6 +122,7 @@
     if($uPracovni) {
       $xtpl->assign([
         'avatar'  =>  $uPracovni->avatar(),
+        'id'      =>  $uPracovni->id(),
```

Comment:
Nevidím z hlavy, jestli toto přidává id do tabulky (to byla dohoda) nebo do záhlaví.